src/common/reducer.js

```diff
@@ -156,6 +156,14 @@ function savePosition(state = {}, action) {
       return state;
   }
 }
+function scrollPosition(state = 0, action) {
```

Comment:
Another function keyword which can be changed to arrow body function.

Reply:
fixed